### PR TITLE
Feat/config data source 

### PR DESCRIPTION
**Pull request by beatrizmonteiromendes**
**Why?**
For the reason that we added a new method AccessBioConfigDataSource for the client to pass the JSON information we needed to update the sample

**Changes**
Create and developed a new class UnicoConfig

**How to test**
xcode

**Preview**
n/a

**Issues**
https://unicoidtech.atlassian.net/jira/software/c/projects/CM/boards/78?modal=detail&selectedIssue=CM-556

### CODE REVIEW

File: unicocheck-sample-swift/UnicoSampleView.swift [hunk 1]
```diff
@@ -56,6 +56,13 @@ class UnicoSampleView: UIViewController, AcessoBioManagerDelegate, SelfieCameraD
...
          Para utlizar o Prova de Vidas com Interação:
          */
+        
+        /// with AcessoBioConfigDataSource implementation
+        //  unicoCheck.build().prepareSelfieCamera(self, config: UnicoConfig())
+
+        // or
+
+        /// with JSON config
         unicoCheck.build().prepareSelfieCamera(self, jsonConfigName: "")
```
Comment (MatheusDomingos) [inline unicocheck-sample-swift/UnicoSampleView.swift:66]: Comment this method.

File: unicocheck-sample-swift/UnicoSampleView.swift [hunk 2]
```diff
@@ -56,6 +56,13 @@ class UnicoSampleView: UIViewController, AcessoBioManagerDelegate, SelfieCameraD
          
          Para utlizar o Prova de Vidas com Interação:
          */
+        
+        /// with AcessoBioConfigDataSource implementation
+        //  unicoCheck.build().prepareSelfieCamera(self, config: UnicoConfig())
```
Comment (MatheusDomingos) [inline unicocheck-sample-swift/UnicoSampleView.swift:61]: Uncomment this line. The main way to implements should be with the Config object

**Review by MatheusDomingos — CHANGES_REQUESTED** [review]

File: unicocheck-sample-swift/UnicoSampleView.swift (hunk 1 above)
Comment (beatrizmonteiromendes) [reply unicocheck-sample-swift/UnicoSampleView.swift:66]: It's not better let one without comment like to show that is how client normally use and now have another one if we want to change?

File: unicocheck-sample-swift/UnicoSampleView.swift (hunk 2 above)
Comment (beatrizmonteiromendes) [reply unicocheck-sample-swift/UnicoSampleView.swift:61]: Okay, got you

**Review by MatheusDomingos — APPROVED** [review]

**Review by Diniz-Lucas — APPROVED** [review]

**Pull request merged**